dashboard/src/app/factories/load-factory/load-factory.controller.ts

```diff
@@ -189,7 +189,7 @@ export class LoadFactoryCtrl {
    */
   startWorkspace(workspace) {
     this.workspace = workspace;
-    var bus = this.cheAPI.getWebsocket().getBus(workspace.id);
```

Comment:
This will not work 
WebSocket class is storing connections per workspace id so WebSocket class has to be updated as well